example/server

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env perl6
+
+use Stomp::Server;
+
+sub MAIN(:$port = 61613, :$host = 'localhost') {
+    my $server = Stomp::Server.new(:$port, :$host);
+
+    my $all-messages = Supplier.new;
+    react {
+        whenever $server.listen -> $v {
+            whenever $v.sent-messages -> $m {
```

Comment:
I may go with something like ```published-messages``` so there is a clearer implication that we may be keeping messages back (say within a transaction, or some authz thing,)